rats-tls/README.md

```diff
@@ -78,6 +85,13 @@ cd /usr/share/rats-tls/samples
 ./rats-tls-server
 ```
 
+**Notice: special prerequisites for SEV remote attestation in software capability.**
+
+- software capability:
+  - [Kernel supports to get guest firmware handle in guest](../hack/sev-attestation/Linux-5.14-kvm-add-hypercall-to-get-sev-guest-firmware-handle.patch).
+  - [The ccp driver supports to get attestation report with `ioctl` command](../hack/sev-attestation/Linux-5.14-drivers-crypto-ccp-add-SEV_GET_REPORT-command.patch).
+  - Start the [attestation evidence broker](../attestation-evidence-broker/README.md) service in host.
```

Comment:
sev attester may run in client side, so it is not appropriate to document the steps in "Run RATS TLS server".